### PR TITLE
Auto accept bid while user in wallet app.

### PR DESCRIPTION
**Pull request by vithanichandresh**
When was user is in wallet app press connect button that time talk button automatically perform now i handle that by this dialog.

### CODE REVIEW

File: lib/infrastructure/providers/my_user_provider/my_user_page_view_model.dart [hunk 1]
```diff
@@ -28,7 +30,56 @@ class MyUserPageViewModel {
...
+      }
+    }
+    return addressOfUserB;
+  }
+
+  Future acceptBid(List<BidIn> bidIns, BuildContext context) async {
+    String? addressOfUserB;
+    BidIn bidIn = bidIns.first;
+    if (!(bidIn.public.active)) {
+      CustomAlertWidget.showToastMessage(context, "Bidder is not active");
```
Comment (2i2i) [inline lib/infrastructure/providers/my_user_provider/my_user_page_view_model.dart:52]: this should never happen ~ we should never be showing non-active bid to Host
but, ok, can leave for now

File: lib/infrastructure/providers/my_user_provider/my_user_page_view_model.dart [hunk 2]
```diff
@@ -38,60 +89,41 @@ class MyUserPageViewModel {
...
 
-    if (!bidIn.public.active) return false;
+    bool camera = true;
+    bool microphone = true;
 
-    if (bidIn.user!.isInMeeting()) {
-      await cancelNoShow(bidIn: bidIn);
-      return false;
+    if (!kIsWeb) {
+      camera = await Permission.camera.request().isGranted;
```
Comment (2i2i) [inline lib/infrastructure/providers/my_user_provider/my_user_page_view_model.dart:103]: what if user does not want to share camera ~ we allow muting, so we should not require permission for camera to start
solve later

Comment (vithanichandresh) [reply lib/infrastructure/providers/my_user_provider/my_user_page_view_model.dart:103]: Then we should have to think of scenario that 

if user pay host for video call and host turn off video then?

Comment (2i2i) [reply lib/infrastructure/providers/my_user_provider/my_user_page_view_model.dart:103]: Host can always turn video off...e.g. by putting tape over their camera
If a feature is unavoidable better to make it easy

Ringing page needs to have audio or video muting as well... Does it not?

**Pull request merged**